digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/AssetFiller.scala

```diff
@@ -8,7 +8,7 @@ import fi.liikennevirasto.digiroad2.asset.SideCode.BothDirections
 import org.joda.time.DateTime
 
 class AssetFiller {
-  val AllowedTolerance = 0.5
+  val AllowedTolerance = 2.0
```

Comment:
LaneFillerissä AllowedTolerance näyttäis olevan vielä 0.5. Pitäisikö sekin muuttaa tähän uuteen arvoon?

Reply:
Nyt kun aloin miettimään niin voisin tässä samalla myös kaistojen osalta irroittaa samuutuksesta muutama korjaus operaatio joko haun tai tallennuksen yhteyteen. Ainakin samoilla arvoilla olevien lisäkaistojen yhdistäminen ja liian lyhyiden pätkien pudottaminen.